pages/_app.js

```diff
@@ -17,13 +19,19 @@ export default function App({ Component, pageProps }) {
       }
     });
     setSpots(newSpotArray);
+    console.log(spots);
   }
 
   return (
     <>
       <GlobalStyle />
 
-      <Component {...pageProps} spots={spots} toggleBookmark={toggleBookmark} />
+      <Component
+        {...pageProps}
+        spots={spots}
+        setSpots={addSpotHandler}
```

Comment:
For less confusing when handling your props I would recommend to use the name `addSpotHandler` and not `setSpots`